remove ebnf images in set operator (#17466)
pingcap/docs: functions-and-operators/set-operators.md

## Use `ORDER BY` and `LIMIT`

> TiDB supports using `ORDER BY` and `LIMIT` clause in set operations. These two clauses must be at the end of the entire statement.

Reviewer: ```suggestion
TiDB supports using `ORDER BY` or `LIMIT` clause on the entire result of set operations. These two clauses must be at the end of the entire statement.
```
